app/components/no_results.js

```diff
@@ -66,15 +68,24 @@ const getStyleFromTheme = makeStyleSheetFromTheme((theme) => {
             paddingHorizontal: 15,
         },
         title: {
-            color: changeOpacity(theme.centerChannelColor, 0.4),
+            color: changeOpacity(theme.centerChannelColor, 0.8),
             fontSize: 20,
-            fontWeight: '600',
+            fontWeight: 'bold',
             marginVertical: 15,
         },
         description: {
-            color: changeOpacity(theme.centerChannelColor, 0.4),
+            color: changeOpacity(theme.centerChannelColor, 0.8),
             fontSize: 17,
```

Comment:
The font size for the description text is `16` in the Figma specs

Reply:
Done